src/js/components/DataChart/utils.js

```diff
@@ -18,21 +18,21 @@ export const heightYGranularity = {
   xlarge: { fine: 11, medium: 5 },
 };
 
-export const halfPad = {
-  xlarge: 'large',
-  large: 'medium',
-  medium: 'small',
-  small: 'xsmall',
-  xsmall: 'xxsmall',
-};
+export const halfPad = (theme) => ({
+  xlarge: theme.dataChart?.halfPad?.xlarge || 'large',
+  large: theme.dataChart?.halfPad?.large || 'medium',
+  medium: theme.dataChart?.halfPad?.medium || 'small',
+  small: theme.dataChart?.halfPad?.small || 'xsmall',
+  xsmall: theme.dataChart?.halfPad?.xsmall || 'xxsmall',
+});
 
-export const doublePad = {
-  large: 'xlarge',
-  medium: 'large',
-  small: 'medium',
-  xsmall: 'small',
-  xxsmall: 'xsmall',
-};
+export const doublePad = (theme) => ({
```

Comment:
@taysea Im not seeing that we are importing doublePad or where we are using this

Reply:
yes same not sure if we need these

Reply:
You're right, I don't see `doublePad` used anywhere, so we can omit any changes to it